hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/codecs/ConsenusServiceStateTranslatorTest.java

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.consensus.impl.test.codecs;
+
+import static com.hedera.node.app.service.consensus.impl.ConsensusServiceImpl.TOPICS_KEY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.hedera.hapi.node.base.Key;
+import com.hedera.hapi.node.base.TopicID;
+import com.hedera.hapi.node.state.consensus.Topic;
+import com.hedera.node.app.service.consensus.impl.codecs.ConsensusServiceStateTranslator;
+import com.hedera.node.app.service.consensus.impl.test.handlers.ConsensusTestBase;
+import com.hedera.node.app.service.mono.legacy.core.jproto.JKey;
+import com.hedera.node.app.spi.fixtures.state.MapWritableKVState;
+import com.hedera.node.app.spi.state.WritableKVState;
+import com.swirlds.merkle.map.MerkleMap;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ConsenusServiceStateTranslatorTest extends ConsensusTestBase {
+
+    @BeforeEach
+    void setUp() {}
+
+    @Test
+    void createMerkleTopicFromTopic() {
+        final var existingTopic = readableStore.getTopic(topicId);
+        assertFalse(existingTopic.deleted());
+
+        final com.hedera.node.app.service.mono.state.merkle.MerkleTopic convertedTopic =
+                ConsensusServiceStateTranslator.pbjToState(topic);
+
+        assertEquals(convertedTopic.getMemo(), getExpectedMonoTopic().getMemo());
```

Comment:
These assertions are repeated a lot.
Would it make sense to have a method:
`assertMatch(com.hedera.node.app.service.mono.state.merkle.MerkleTopic merkleTopic, Topic pbjTopic)`
and that method includes all of the individual property assertions?

Reply:
sure no problem I will do so.